explorer/gen_rtti.py

```diff
@@ -179,7 +179,20 @@ class \s+
                  ;$"""
 
 
+def usage() -> None:
+    print("Usage:")
+    print(f"  {sys.argv[0]} <input_filename> <header_filename> <cpp_filename> "
+           "<relative_header>")
+    print()
+    print("For more information read this script's description within its "
+          "content.")
+
+
 def main() -> None:
+    if len(sys.argv) != 5:
+        usage()
+        exit(1)
+
```

Comment:
Let's just add documentation to the top if you want to document syntax. This script isn't intended to be run directly -- it should be run through the //explorer:gen_rtti build target.

Really we should probably just remove the python3 line from the top so that it's not run directly by accident.

Reply:
Removing the python3 line from the top way sounds better, let me update this PR.

Reply:
Hmmm, looks like there is a check to verify executables have shebangs:

```
 check that executables have shebangs.....................................Failed
```

I'll update this PR to specify this script is not intended to be run directly in the docstring.